spdockit-spo/office-365-groups-and-onedrive/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Office 365 Groups and Teams  
+description: This article describes the available SPDocKit Management actions and wizards for Office 365 Groups and Teams.
+author: Tomislav Sirovec
+date: 14/11/2017
+---
+
+This article describes the Groups and Teams general report, as well as the report for each group in particular.  
+Also, it describes available SPDocKit Management actions and wizards for Office 365 Groups and Teams.  
+Additionally it provides information on our OneDrive report.
+
+__Note!__ Before you can start exploring Groups and Teams reports, SPDocKit will ask you to provide permissions for the following:  
+- __View your basic profile__ (allows the app to see your basic profile.     Name, picture, username).
+- __Access your data anytime__ (allows the app to see and update your data, even when you are not currently using the app).
+- __Sign in as you__ (allows you to sign into the app with your account and allows the app to read your basic profile information).
+- __Read directory data__ (allows the app to read data in your organization's directory).
+- __Read and write all groups__ (allows the app to create groups and read all group properties and memberships on your behalf. Additionally allows the app to manage your groups and to update group content for groups you are a member of).
+- __Read and write all users' full profiles__ (allows the app to read and write the full set of profile properties, reports, and managers of other users in your organizatios, on your behalf).
+
```

Comment:
Add note on what will happen if I do not accept or do not have right privileges to accept this.